### ISSUE TITLE
Aria grouping roles should not receive focus

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-core#6]
With new mcq and gmcq, the options are grouped with a parent having `[role=group]` or `[role=radiogroup]` and `[aria-labelledby]`. 

Adapt's focus functions will automatically focus on `[aria-labelledby]` elements when ask to `focusFirst` or `focusNext` due to this [configuration option](https://github.com/adaptlearning/adapt-contrib-core/blob/0dacbfbe0215c452c9b2bc4a96db44c38f37a6b9/js/a11y.js#L42).

As the container receives focus, with a screen reader, the entire mcq or gmcq will read once. Then upon keyboard navigation the mcq will read again.

It should be constrained to exclude something like `[aria-labelledby]:not([role=radiogroup],[role=group],[role=tablist],[role=list],[role=tree],[role=treegrid],[role=table],[role=grid],[role=menu],[role=rowgroup])`, explicitly preventing Adapt from focusing on grouping roles.

Reference https://www.w3.org/TR/wai-aria-1.1/#role_definitions
